addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/toberemoved/cache/ExpiringCacheAsync.java

```diff
@@ -6,7 +6,7 @@
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
  */
```

Comment:
@davidgraeff what is the status, I believe the class should still be removed, wdyt?

Reply:
It's now part of ESH. But the API changed to java futures, so some code writing would be necessary, too.

Reply:
Will you pick it up? Otherwise @cweitkamp might be interested?

Reply:
Allways. Thanks for asking. Let @davidgraeff decide. One of us can fix and contribute.

Reply:
> I don't have time right now to be honest

@cweitkamp I think that is a yes

Reply:
I'm on it. I guess this PR will be obsolete with my changes.